Openshift-EE/pipelines/Prometheus/stages/3-functional/ELHN-deploy-prometheus/prometheus-provision

```diff
@@ -37,24 +46,33 @@ run_test=apps/prometheus/deployers/run_litmus_test.yml
 
 cat $run_test
 
-
+# Run the Litmus job and get the details of the litmus job from litmus_job_runner utils.
 bash ../Openshift-EE/utils/litmus_job_runner label='app:prometheus-deployment-litmus' job=$run_test
 
 echo "Dumping state of cluster post job run"; echo ""
 cd ..
+# Get the cluster state Once the litmus jobs completed.
 bash Openshift-EE/utils/dump_cluster_state;
+# Update the e2e event for the job.
 bash Openshift-EE/utils/event_updater jobname:prometheus-provision $test_name jobid:"$job_id" pipelineid:"$pipeline_id" testcaseid:"$case_id"
 
-
+# Update result of the test case in github mayadata-io/e2e-openshift repository.
 if [ "$?" != "0" ]; then
 #python3 Openshift-EE/utils/result/result_update.py $job_id ELHN 3-functional "Deploying prometheus application" Fail $pipeline_id "$current_time" $commit_id $gittoken
 exit 1;
 fi
 
+################
+# LitmusBook 2 #
+################
 
 echo "*******Deploying Prometheus Loadgen Application****"
 #kubectl apply -f Openshift-EE/utils/result-loadgen-provision.yml
 
+#testcase name for the prometheus loadgen will be appended with busybox-{{ action }}- {{ app-namespace }}.
```

Comment:
@harshita-sharma011 is this Prometheus or busybox loadgen correct the type for test case name.